app/database/queries/breeze_buddy/knowledge_base.py

```
@@ -551,3 +551,22 @@ def hybrid_search_chunks_query(
+
+    The age floor debounces the poller (Sheets bumps modifiedTime on every
+    keystroke-ish save; we never re-sync a sheet more often than this).
+    """
+    text = f"""
+        SELECT * FROM "{KB_DOCUMENT_TABLE}"
+        WHERE "source_type" = 'google_sheet'
+          AND "status" = 'READY'
+          AND ("synced_at" IS NULL
+               OR "synced_at" < now() - make_interval(secs => $1))
+        ORDER BY "synced_at" NULLS FIRST;
+    """
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)